extensions/wikia/RTE/RTE.class.php

```diff
@@ -92,7 +95,7 @@ public static function renderPlaceholder($label, $data) {
 	 *
 	 * @author Inez KorczyDski, Macbre
 	 */
-	public static function init(&$form) {
+	public static function init( &$form ) {
 		global $wgOut, $wgHooks, $wgAllInOne, $wgRequest;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)